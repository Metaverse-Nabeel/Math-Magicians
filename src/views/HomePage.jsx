const HomePage = () => (
  <>
    <div className="home">
      <h1 className="homeTitle">Welcome to Math Magicians!</h1>
      <p className="homeText">
        Welcome to Math Magicians, your ultimate destination for all things
        mathematics! Whether you&apos;re a seasoned mathematician or just
        getting started, our website is here to ignite your passion for numbers
        and equations. Explore the fascinating world of mathematics through our
        informative articles, engaging tutorials, and mind-boggling puzzles.
      </p>
      <p className="homeText">
        Discover the beauty of prime numbers, unravel the secrets of geometry,
        and delve into the depths of calculus. From algebraic equations to
        trigonometric functions, we&apos;ve got you covered.
      </p>
      <p className="homeText">
        Looking to enhance your problem-solving skills? Check out our collection
        of challenging math problems and put your logical thinking to the test.
        Our step-by-step solutions will guide you towards mathematical mastery.
      </p>
      <p className="homeText">
        Join our vibrant community of math enthusiasts in our interactive
        forums. Connect with fellow mathematicians, exchange ideas, and discuss
        the latest developments in the world of mathematics.
      </p>
      <p className="homeText">
        Whether you&apos;re a student seeking homework help or a curious mind
        hungry for knowledge, Math Magicians is your portal to mathematical
        enlightenment. Prepare to be captivated by the wonders of numbers and
        experience the magic of mathematics like never before.
      </p>
      <p className="homeText">
        Get ready to unlock the infinite possibilities that mathematics offers.
        Dive into our world of numbers and equations, and let Math Magicians be
        your guide. Get ready to witness the magic of mathematics unfold before
        your eyes!
      </p>
    </div>
  </>
);
export default HomePage;
