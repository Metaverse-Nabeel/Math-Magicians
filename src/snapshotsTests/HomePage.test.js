import React from 'react';
import { render } from '@testing-library/react';
import HomePage from '../views/HomePage';

describe('Tests for the Home page', () => {
  it('Should render the Home page properly', () => {
    const homePage = render(<HomePage />);
    expect(homePage).toMatchSnapshot();
  });
});
