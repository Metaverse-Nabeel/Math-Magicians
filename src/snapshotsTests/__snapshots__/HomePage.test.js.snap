// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for the Home page Should render the Home page properly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="home"
      >
        <h1
          class="homeTitle"
        >
          Welcome to Math Magicians!
        </h1>
        <p
          class="homeText"
        >
          Welcome to Math Magicians, your ultimate destination for all things mathematics! Whether you're a seasoned mathematician or just getting started, our website is here to ignite your passion for numbers and equations. Explore the fascinating world of mathematics through our informative articles, engaging tutorials, and mind-boggling puzzles.
        </p>
        <p
          class="homeText"
        >
          Discover the beauty of prime numbers, unravel the secrets of geometry, and delve into the depths of calculus. From algebraic equations to trigonometric functions, we've got you covered.
        </p>
        <p
          class="homeText"
        >
          Looking to enhance your problem-solving skills? Check out our collection of challenging math problems and put your logical thinking to the test. Our step-by-step solutions will guide you towards mathematical mastery.
        </p>
        <p
          class="homeText"
        >
          Join our vibrant community of math enthusiasts in our interactive forums. Connect with fellow mathematicians, exchange ideas, and discuss the latest developments in the world of mathematics.
        </p>
        <p
          class="homeText"
        >
          Whether you're a student seeking homework help or a curious mind hungry for knowledge, Math Magicians is your portal to mathematical enlightenment. Prepare to be captivated by the wonders of numbers and experience the magic of mathematics like never before.
        </p>
        <p
          class="homeText"
        >
          Get ready to unlock the infinite possibilities that mathematics offers. Dive into our world of numbers and equations, and let Math Magicians be your guide. Get ready to witness the magic of mathematics unfold before your eyes!
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="home"
    >
      <h1
        class="homeTitle"
      >
        Welcome to Math Magicians!
      </h1>
      <p
        class="homeText"
      >
        Welcome to Math Magicians, your ultimate destination for all things mathematics! Whether you're a seasoned mathematician or just getting started, our website is here to ignite your passion for numbers and equations. Explore the fascinating world of mathematics through our informative articles, engaging tutorials, and mind-boggling puzzles.
      </p>
      <p
        class="homeText"
      >
        Discover the beauty of prime numbers, unravel the secrets of geometry, and delve into the depths of calculus. From algebraic equations to trigonometric functions, we've got you covered.
      </p>
      <p
        class="homeText"
      >
        Looking to enhance your problem-solving skills? Check out our collection of challenging math problems and put your logical thinking to the test. Our step-by-step solutions will guide you towards mathematical mastery.
      </p>
      <p
        class="homeText"
      >
        Join our vibrant community of math enthusiasts in our interactive forums. Connect with fellow mathematicians, exchange ideas, and discuss the latest developments in the world of mathematics.
      </p>
      <p
        class="homeText"
      >
        Whether you're a student seeking homework help or a curious mind hungry for knowledge, Math Magicians is your portal to mathematical enlightenment. Prepare to be captivated by the wonders of numbers and experience the magic of mathematics like never before.
      </p>
      <p
        class="homeText"
      >
        Get ready to unlock the infinite possibilities that mathematics offers. Dive into our world of numbers and equations, and let Math Magicians be your guide. Get ready to witness the magic of mathematics unfold before your eyes!
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
